daemon/database/artistrepo.py

```diff
@@ -1,14 +1,16 @@
 import logging
 
+from decorator.injectdbsession import inject_db_session
+
 from .entity import Artist
-from .connection import Connection
 
+
+@inject_db_session('_session')
```

Comment:
I've seen your implementation of the decorator has the default value, I think the `_session` can be removed.